docpipeline/prepare.py

```diff
@@ -23,9 +23,5 @@ def add_prettyprint(output_path):
         html = html.replace(
             '<code class="lang-'.encode(), '<code class="prettyprint lang-'.encode()
```

Comment:
`'<code class="lang-'.encode(), '<code class="prettyprint lang-'.encode()`
Unrelated to this PR, should we also look into changing this?

Reply:
You mean add `prettyprint` to the `<pre>` tag so that we can delete the extra step in the templates?

Reply:
Not delete the extra steps in the templates, those are fine to leave. I believe this was added to accommodate for extra code blocks that get generated but don't get `prettyprint` added, however since we moved from adding these classes to `<pre>` blocks I was wondering if we should move them to `<pre>` instead of `<code>`.